src/sentry/event_manager.py

```diff
@@ -190,6 +190,10 @@ def plugin_is_regression(group, event):
     return True
 
 
+class DiscardedHash(Exception):
```

Comment:
I have zero objective reasons to justify this, but I think `HashDiscarded` sounds better 🙃